src/GlobalStyles.js

```diff
@@ -12,9 +12,15 @@ body {
   background-color: var(--bg-color-main);
 }
 
+header {
+  background-color: var(--bg-color-header);
+  text-align: center;
+  padding: 10px;
+}
+
 main {
   width: 360px;
-  padding: 50px 0 20px;
+  padding: 40px 0 20px;
   margin: 0 auto;
   overflow-y: auto;
 }
```

Comment:
I wouldn't put these styles in the GlobalStyles. Especially for the header: if you have another header somewhere in your app, the same styles will be applied. Why not outsource them as just styled components as we did with the `Button` in the Skorekeeper App? I think that would be a bit cleaner than defining the styles here as they are pretty specific.

Also, if I'd be a new developer working on this project with you, I would probably be a bit confused of the fact that the components do have styles, but are not styled-components. It's a bit inconsistent and usually GlobalStyles are just for resetting CSS styles and not defining styles for components.

Reply:
I changed it, looks much cleaner now. Thanks.